extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java

```diff
@@ -349,4 +350,72 @@ protected long getTimestampSeconds(String regexPattern) throws ParsingException
             return 0;
         }
     }
+
+    /**
+     * The host of the stream (Eg. peertube.cpy.re).
+     * If the host is not available, or if the service doesn't use
+     * a federated system, but a centralised system,
+     * you can simply return an empty string.
+     * @return the host of the stream or an empty String.
+     * @throws ParsingException
+     */
+    @Nonnull
+    public abstract String getHost() throws ParsingException;
+
+    /**
+     * The privacy of the stream (Eg. Public, Private, Unlisted…).
+     * If the privacy is not available you can simply return an empty string.
+     * @return the privacy of the stream or an empty String.
+     * @throws ParsingException
+     */
+    @Nonnull
+    public abstract String getPrivacy() throws ParsingException;
```

Comment:
I know, that PeerTube calls this privacy, but from my understanding, visibility or availability are better terms here, because both describe what impact the value of this method has.
@Stypox @B0pol What do you think?

Reply:
Even YouTube calls it privacy [Change video privacy settings — YouTube support](https://support.google.com/youtube/answer/157177)
I think if we should rename the method, it should be for availability.

Reply:
Oh. Ok. In this case, only the German translation is a bit different here. I am fine with privacy :+1: